site/src/theme/dark/branding.ts

```diff
@@ -1,7 +1,7 @@
 import type { Branding } from "../branding";
 import colors from "../tailwindColors";
 
-export default {
```

Comment:
Forgot to leave a comment on these earlier, but the reason why I switched the syntax from `satisfies` to type annotations is because when I was adding additional properties to the type definitions, `satisfies` wasn't always immediately kicking up an error when something was only partially updated, and it made way more red squiggles than it needed to